src/components/EasyPill/EasyPill.module.css

```diff
@@ -2,3 +2,11 @@
   align-self: center;
   margin-right: var(--rvr-space-md);
 }
+
+.tooltipText {
+  height: var(--rvr-line-height-lg);
```

Comment:
is the tooltip height the same as the line height? would the element height not just flow with the content?

Reply:
Yeah, that's right, I will change it.